src/osd/PG.cc

```diff
@@ -4509,14 +4541,16 @@ void PG::chunky_scrub(ThreadPool::TPHandle &handle)
 	   * left end of the range if we are a tier because they may legitimately
 	   * not exist (see _scrub).
 	   */
-	  int min = MAX(3, cct->_conf->osd_scrub_chunk_min);
+	  int min = MAX(3, cct->_conf->osd_scrub_chunk_min /
+			scrubber.preempt_divisor);
+	  int max = MAX(min, cct->_conf->osd_scrub_chunk_max);
```

Comment:
Why are we adjusting the max as well as the min? It’s not like the min can be larger with the changes we’ve made.

Reply:
just make sure we tolerate a bad config value.  (this is just moving the line up from below unchanged to tidy the code, btw)